product/components/ProductCard.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import faker from "faker";
+
+import ProductCard from "./ProductCard";
+
+export default {title: "ProductCard"};
+
+export const noData = () => (
+  <ProductCard
+    add={() => {}}
+    product={{
+      id: faker.random.uuid(),
+      category: faker.commerce.department(),
+      image: faker.image.food(),
+      description: faker.lorem.paragraph(),
+      title: faker.commerce.productName(),
+      price: faker.commerce.price(),
```

Comment:
```suggestion
      price: Number(faker.commerce.price()),
```
Por que devuelve string